src/dcos_e2e/cluster.py

```diff
@@ -116,9 +117,14 @@ def wait_for_dcos(self) -> None:
             '--diag',
         ]
 
+        # Must be run privileged
```

Comment:
I don't like this change, it seems hidden but I'm happy to discuss. Why not just make the user use `sudo` where appropriate?